act-rules-format.bs

```diff
@@ -296,6 +296,19 @@ Even concepts like headings and images can be misunderstood. These terms could r
   </blockquote>
 </div>
 
+<div class=example>
+  <p>**Example:** The applicability of a rule with the page as a test target</a></p>
+  <blockquote>
+    <p>The rule applies to any page where the root element is an `html` element, and which is not embedded in another page.</p>
```

Comment:
While both "root" and "element" are defined in DOM (https://www.w3.org/TR/dom/#concept-tree-root and https://www.w3.org/TR/dom/#concept-element), I don't think we can use "root element" unambiguously. The term "document element" seems like a better fit: https://www.w3.org/TR/dom/#document-element